tests/contrib/test_gce.py

```diff
@@ -14,102 +14,195 @@
 
 """Unit tests for oauth2client.contrib.gce."""
 
-import json
 from six.moves import http_client
-from six.moves import urllib
 import unittest2
-
+import time
 import mock
+import json
```

Comment:
1. Please move `time` and `json` above these imports (they are in the stdlib, none of the rest are)
1. Please put imports in alphabetical order.